src/pkgcheck/addons/git.py

```diff
@@ -79,14 +76,37 @@ def __init__(self, *args, commit):
         self.commit = commit
 
 
+class GitConfig:
+    """Manages temporary file which holds git config for disabling
+    safe directory feature of git."""
+
+    def __init__(self):
+        fd, self.path = tempfile.mkstemp()
+        print('AAA', self.path)
```

Comment:
?

Reply:
Looks like debugging :-)

Reply:
Oh, I missed this line.
Yes, this was for debugging the auto delete part.